src/lib/Group.ts

```diff
@@ -203,6 +203,13 @@ export class Group<
     return XMTP.isGroupActive(this.client.address, this.id)
   }
 
+  async addedByAddress(skipSync = false): Promise<string> {
+    if (!skipSync) {
+      await this.sync()
+    }
+    return XMTP.addedByAddress(this.client.address, this.id)
```

Comment:
I know this is a draft PR, just trying to add actionable steps forward:
1. The addedByAddress will need to be added in src/index.ts
2. Then it will need to be bridged over in android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt & ios/XMTPModule.swift
3. Both of which will require bumps on gradle/pod versions for your changes